Load/ontology/General/collections/collections.yaml

```diff
@@ -11,93 +11,147 @@ stable_id: EUPATH_0009251
+isProportion: true
+isCompositional: true
+normalizationMethod: sumToUnity
 ---
 stable_id: EUPATH_0009254
 member: taxon
 memberPlural: taxa
 label: relative abundance of order data
+isProportion: true
+isCompositional: true
+normalizationMethod: sumToUnity
 ---
 stable_id: EUPATH_0009255
 member: taxon
 memberPlural: taxa
 label: relative abundance of family data
+isProportion: true
+isCompositional: true
+normalizationMethod: sumToUnity
 ---
 stable_id: EUPATH_0009256
 member: taxon
 memberPlural: taxa
 label: relative abundance of genus data
+isProportion: true
+isCompositional: true
+normalizationMethod: sumToUnity
 ---
 stable_id: EUPATH_0009257
 member: taxon
 memberPlural: taxa
 label: relative abundance of species data
+isProportion: true
+isCompositional: true
+normalizationMethod: sumToUnity
 ---
 stable_id: EUPATH_0009351
 member: taxon
 memberPlural: taxa
 label: absolute abundance of kingdom data
+isProportion: false
+isCompositional: true
+normalizationMethod: NULL
 ---
 stable_id: EUPATH_0009352
 member: taxon
 memberPlural: taxa
 label: absolute abundance of phylum data
+isProportion: false
+isCompositional: true
+normalizationMethod: NULL
 ---
 stable_id: EUPATH_0009353
 member: taxon
 memberPlural: taxa
 label: absolute abundance of class data
+isProportion: false
+isCompositional: true
+normalizationMethod: NULL
 ---
 stable_id: EUPATH_0009354
 member: taxon
 memberPlural: taxa
 label: absolute abundance of order data
+isProportion: false
+isCompositional: true
+normalizationMethod: NULL
 ---
 stable_id: EUPATH_0009355
 member: taxon
 memberPlural: taxa
 label: absolute abundance of family data
+isProportion: false
+isCompositional: true
+normalizationMethod: NULL
 ---
 stable_id: EUPATH_0009357
 member: taxon
 memberPlural: taxa
 label: absolute abundance of species data
+isProportion: false
+isCompositional: true
+normalizationMethod: NULL
 ---
 stable_id: EUPATH_0009356
 member: taxon
 memberPlural: taxa
 label: absolute abundance of genus data
+isProportion: false
+isCompositional: true
+normalizationMethod: NULL
 ---
 stable_id: EUPATH_0009247
 member: gene
 memberPlural: genes
 label: 4th level EC metagenome abundance data
+isProportion: false
+isCompositional: true
+normalizationMethod: RPK
 ---
 stable_id: EUPATH_0009248
 member: pathway
 memberPlural: pathways
 label: metagenome enzyme pathway abundance data
+isProportion: false
+isCompositional: true
+normalizationMethod: RPK
 ---
 stable_id: EUPATH_0009249
 member: pathway
 memberPlural: pathways
 label: metagenome enzyme pathway coverage data
+isProportion: false
+isCompositional: false
+normalizationMethod: NULL
 # ---
 # stable_id: EUPATH_0009268
 # member: taxon
 # memberPlural: taxa
 # label: fungal taxon detected by sequence match
 ---
 stable_id: EUPATH_0009269
-member: match
-memberPlural: matches
+member: taxon
+memberPlural: taxa
 label: normalized number of taxon-specific sequence matches
+isProportion: false
+isCompositional: true
```

Comment:
is this always under corral? cause if so, i think id put isCompositional = false. the vast majority of reads wont be for eukaryotes, and id think a measure that considered a small minority of reads shouldnt be considered compositional any longer.